README.md

```diff
@@ -128,6 +128,109 @@ NOTE: You can leave that static server running in one session while you continue
 to execute `tox` on another session. That server will pick up any new changes to
 the report automatically so you don't have to restart it each time.
 
+Troubleshooting
+----------
+
+Since Kubernetes charms are not supported by the `juju debug-hooks`, the only
+way to intercept the control of code execution is to initialize the non-tty-bound
+debugger session and connect to the session externally.
+
+For this purpose, we chose the [rpdb](https://pypi.org/project/rpdb/), the
+remote Python debugger based on pdb.
+
+So, given you have already deployed your model and you would like to debug the
+`config-changed` hook, you should do the following:
+
+Given `prometheus` is an application name and `lma` is a Juju model name, 
+execute the following command to grab an interactive Bash shell to the operator
+pod:
+
+
+    kubectl exec -it pod/prometheus-operator-0 -n lma -- /bin/sh
+
+
+Install the editor and the RPDB:
+
+
+    apt update
+    apt install telnet vim -y
+    pip3 install rpdb
+
+
+Open the charm entry point in the editor:
+
+
+    vim /var/lib/juju/agents/unit-prometheus-0/charm/src/charm.py
+
+
+Find a `on_config_changed_handler` function definition in the `charm.py` file.
+Modify it as follows:
+
+
+    def on_config_changed_handler(event, fw_adapter):
+        import rpdb
+        rpdb.set_trace()
+        # < ... rest of the code ... >
+
+
+Save the file (`:wq`). Do not close the current shell session!
+
+Open another terminal session to the machine with Juju client and trigger the
```

Comment:
Can we just say "Open another terminal session in your workstation and trigger the `config-changed` hook as follows:"